### ISSUE TITLE
Split js and css to their own files

### ISSUE THREAD

**Issue by sakisv** [sakisv/canibeloud#5]
The entire HTML, JS and CSS are hardcoded as a raw string. They should be split to their own files and managed separately, either using the actix's static-files approach[^1] or by embedding them in the resulting binary[^2][^3].

Some obvious (and symmetrical) pros and cons for each approach:

### Serve them as static files

➕ No need to recompile to test your changes
➕ Smaller binary size
➖ Security implications if the request is not properly sanitized
➖ More things to move around when deploying

### Include them in the binary

➕ Safer - i.e. something either exists in the binary or not, not touching the filesystem
➕ Single binary contains everything
➖ Slower feedback loop, need to recompile to test every change
➖ Larger binary

### Combining them?

Investigate if it's possible to combine the two approaches, i.e. when developing use the first approach, but when we're releasing include the static files in the binary. There are some dangers there like new files not being included, or not commiting the changes, but something worth exploring, even to see if it's possible at all.

[^1]: https://actix.rs/docs/static-files
[^2]: https://stackoverflow.com/a/27140747/1477072
[^3]: https://doc.rust-lang.org/std/macro.include_bytes.html

**Comment by sakisv** [comment]
Closed by #20 

While I did not split js and css to their own files, I did split out the hardcoded HTML to its own file.

Went with the 2nd option and the ➖ bits are aleviated as follows:

#### ➖ Slower feedback loop, need to recompile to test every change

Using [cargo-watch](https://crates.io/crates/cargo-watch) this goes away 

#### ➖ Larger binary

Before:

```
$ ls -alh target/release/canibeloud
-rwxr-xr-x 1 sakis staff 8.5M Feb  4 11:05 target/release/canibeloud
```

After:

```
$ ls -alh target/release/canibeloud
-rwxr-xr-x 1 sakis staff 8.5M Feb  4 11:06 target/release/canibeloud
```

**Comment by sakisv** [comment]
Closed by #20
